pkgs/by-name/tr/trigger-control/package.nix

```diff
@@ -41,6 +44,15 @@ stdenv.mkDerivation rec {
     libdecor
   ];
 
+  patches = [
+    # Fix build on clang https://github.com/Etaash-mathamsetty/trigger-control/pull/23
+    (fetchpatch {
+      name = "clang.patch";
+      url = "https://github.com/Etaash-mathamsetty/trigger-control/pull/23/commits/bbec33296fdac7e2ca0398ae19ca8de8ad883407.patch";
```

Comment:
thanks!
I think the url needs to be:
https://github.com/Etaash-mathamsetty/trigger-control/commit/bbec33296fdac7e2ca0398ae19ca8de8ad883407.patch
as that follows the other patches in nixpkgs, i do not know if the url with  `pull/32/commits` is stable.

Reply:
Done